src/static/helpers/slasHelper.ts

```diff
@@ -111,10 +111,7 @@ export async function authorize(
   // set manual redirect on server since node allows access to the location
   // header and it skips the extra call. In the browser, only the default
   // follow setting allows us to get the url.
-  const fetchOptions = {
-    ...slasClient.clientConfig.fetchOptions,
-    ...(!onClient && {redirect: 'manual'}),
-  };
+  const redirect: RequestRedirect = 'manual';
```

Comment:
This sets `redirect` to `"manual"` regardless of environment, but the comment seems to suggest that the setting in browser must be `"follow"`. What is the intended behavior in browser?

Reply:
Exactly. In node, you can set `manual` and still get a readable Response that includes the location header. In the browser, setting it to `manual` gives you an unreadable Response. You need to leave it on `follow` and then you can use the `url` field. https://stackoverflow.com/questions/39735496/redirect-after-a-fetch-post-call